include/pstl_offload/internal/usm_memory_replacement.h

```diff
@@ -281,6 +317,30 @@ inline void* __attribute__((always_inline)) __libc_realloc(void *__ptr, std::siz
     return realloc(__ptr, __size);
 }
 
+#elif _WIN64
+
+inline void* __attribute__((always_inline)) _aligned_malloc(std::size_t __size, std::size_t __alignment)
+{
+    // _aligned_malloc should reject zero or not power of two alignments
+    if (!::__pstl_offload::__verify_aligned_new_param(__alignment))
+    {
+        return nullptr;
+    }
+    return ::__pstl_offload::__errno_handling_internal_aligned_alloc(__size, __alignment);
+}
+
+inline void* __attribute__((always_inline)) _aligned_realloc(void* __ptr, std::size_t __size, std::size_t __alignment)
+{
+    // _aligned_realloc should reject zero or not power of two alignments, but not when it calls _aligned_free
+    if (__size && !::__pstl_offload::__verify_aligned_new_param(__alignment))
+    {
+        return nullptr;
+    }
+    return ::__pstl_offload::__internal_aligned_realloc(__ptr, __size, __alignment);
+}
+
```

Comment:
Logic here is next: for TBBmalloc no one is asked about those functions for 10+ years, so we may want to ignore them as well. 

From the different side, `_recalloc`/`_aligned_recalloc` is low hanging fruit.

Reply:
I have no strong feelings about what specific functions we support.  I suppose the thing to do is to decide on them and document what is supported so it is not a guessing game for the user.  Perhaps this becomes an issue to be resolved in a later doc specific PR.  This is a fine resolution from my perspective without code changes.

Reply:
Good point. @rarutyun has a table with such functions for Linux, it might be complimented by Windows ones and releases as part of the product's documentation.